### PR TITLE
Add temperature to equations of state.

### PR DESCRIPTION
**Pull request by ejtovar**

### CODE REVIEW

**Comment by tamiko** [comment]
@ejtovar Overall, this looks good. Would you mind to add or update some of the tests? It would be good to have verified test vectors for the temperature in our testsuite.

**Comment by ejtovar** [comment]
@tamiko I have updated the equation of state tests to include temperature now.

File: source/euler_aeos/equation_of_state_polytropic_gas.h [hunk 1]
```diff
@@ -24,6 +24,10 @@ namespace ryujin
       {
         gamma_ = 7. / 5.;
         this->add_parameter("gamma", gamma_, "The ratio of specific heats");
+
+        cv_ = 718.;
```
Comment (tamiko) [inline source/euler_aeos/equation_of_state_polytropic_gas.h:28]: Out of curiosity: cv for an ideal gas is 3/2R. Where does the `718` come from?

**Review by tamiko — APPROVED** [review]

**Comment by tamiko** [comment]
@ejtovar We will need a functionality to output these values next. Let's discuss in a minute.

File: source/euler_aeos/equation_of_state_polytropic_gas.h (hunk 1 above)
Comment (tamiko) [reply source/euler_aeos/equation_of_state_polytropic_gas.h:28]: Let's address this and other constants (and their units) in a follow-up PR.

**Pull request merged**